product-service/serverless.ts

```diff
@@ -76,7 +116,13 @@ const serverlessConfiguration: Serverless = {
           http: {
             method: 'get',
             path: 'products',
-            cors: true
+            cors: true,
+            authorizer: {
+              name: 'cognitoAuthorizer',
+              type: 'COGNITO_USER_POOLS',
+              arn: 'arn:aws:cognito-idp:eu-west-1:714652663732:userpool/eu-west-1_zTPf140Wi',
```

Comment:
Seems like you know it, but better to use AWS::Region and AWS::AccountId here as you've done in importProductsFile.

Reply:
Thanks, forgot to replace here